src/analysis/config.d

```diff
@@ -190,7 +190,7 @@ struct StaticAnalysisConfig
 	string has_public_example = Check.disabled;
 
 	@INI("Check for asserts without an explanatory message")
-	string assert_without_msg = Check.disabled;
+	string assert_without_msg = Check.skipTests;
```

Comment:
I don't agree with this change. Wasn't the check designed for phobos in the first place ?